Super lame.  Drop me once fontconfig learns how to ignore spaces.

diff -up fontconfig-2.8.0/conf.d/30-urw-aliases.conf.jx fontconfig-2.8.0/conf.d/30-urw-aliases.conf
--- fontconfig-2.8.0/conf.d/30-urw-aliases.conf.jx	2009-11-16 12:24:52.000000000 -0500
+++ fontconfig-2.8.0/conf.d/30-urw-aliases.conf	2011-05-31 16:00:06.162549910 -0400
@@ -29,6 +29,10 @@
 	  <family>Zapf Dingbats</family>
 	  <accept><family>Dingbats</family></accept>
 	</alias>
+	<alias binding="same">
+	  <family>ZapfDingbats</family>
+	  <accept><family>Dingbats</family></accept>
+	</alias>
 	<match target="pattern">
 	  <test name="family">
 	    <string>Symbol</string>
